docker/apply-config-from-env-with-prefix.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please let me know about the license header.

Also usually in OSS projects it is good to refer to the original file.
Do you have any guidelines ?
Otherwise I can do it the way we are used to do in the Apache Software Foundation

Reply:
For now the project is under the BSD license, which as far as I understand is equivalent to the Apache License. Actually, I am told that in some arcane details it is even more permissive than Apache, though I am not a lawyer and some of these things fly way over my head...

Anyway, if there's a reason to switching to Apache, we can definitely consider it in the future, but for now, I think it's probably cleaner to not mix licenses.

Regarding headers, I guess that is best practice. We don't have a header in each file currently. We could add one to all files in the project. Might be better to do that in one shot.